cvat/apps/webhooks/models.py

```diff
@@ -0,0 +1,99 @@
+from enum import Enum
```

Comment:
Need to add a license header in all new files. Also need to modify the header in files where it already exists